generators/import-jdl/index.js

```diff
@@ -100,50 +103,22 @@ module.exports = class extends BaseGenerator {
 
             parseJDL() {
                 this.log('The JDL is being parsed.');
+                const jdlImporter = new jhiCore.JDLImporter(this.jdlFiles, {
+                    databaseType: this.prodDatabaseType,
```

Comment:
for app generation these wont be present is that fine?

Reply:
Why?

Reply:
You mean the variables?

Reply:
Oh I get it. It's perfectly fine. The importer will handle it just fine.